quickassist/lookaside/access_layer/src/sample_code/functional/dc/stateful_sample/paper4

```diff
@@ -196,7 +196,7 @@ modes.  They have been omitted from the diagrams to preserve clarity.)
 .sh "Conclusion"
 .pp
 The type of simple modelling system presented above is of interest for a
-number of reasons.  Seen as a programing by example system, 
+number of reasons.  Seen as a programming by example system, 
```

Comment:
This should not be changed as paper4 is a standard text file in the Calgary corpus used for data compression benchmarking. It could potentially alter some test results. (Actually we're removing this file in next release anyway)

Reply:
NO change made.

Reply:
I still see the change?

Reply:
I've reverted that change and re-pushed.